base/host/packages/cuttlefish-base/etc/security/limits.d/1_cuttlefish.conf

```diff
@@ -0,0 +1,4 @@
+# Cuttlefish with GPU passthrough will use additional file descriptors on the
+# host for Gralloc buffers.
+* hard nofile 4096
```

Comment:
Rather than `*` what do you think of using the group `@cvdnetwork`?

From `/etc/security/limits.conf`:
```
#Each line describes a limit for a user in the form:
#
#<domain>        <type>  <item>  <value>
#
#Where:
#<domain> can be:
#        - a user name
#        - a group name, with @group syntax
#        - the wildcard *, for default entry
#        - the wildcard %, can be also used with %group syntax,
#                 for maxlogin limit
```

Reply:
Done!